### PR TITLE
Issue 97 - SubQuery -> SubSquid

### PR DESCRIPTION
**Pull request by poppyseedDev**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Contributes to  #97 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

File: developers/contributing/how-to-start-contributing.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Current Indexers we have/use:
 
 - SubSquid
   - RMRK: [rubick](https://github.com/kodadot/rubick)
-- SubQuery
+- SubSquid
```
Comment (yangwao) [inline developers/contributing/how-to-start-contributing.md:99]: this one is actually SubQuery

**Review by yangwao — APPROVED** [review]
